src/AutomationTests/AutomationIntegrationTests.cs

```diff
@@ -119,39 +120,7 @@ public void Scan_Integration_WpfControlSampler()
             Scan_Integration_Core(WpfControlSamplerAppPath, WpfControlSamplerKnownErrorCount);
         }
 
-        [TestMethod]
-        [Timeout(30000)]
-        public void SingleWindowScan_MultipleRootsEnabledThrows()
```

Comment:
Do we have another place where we're scanning multiple roots? We should have this somewhere...